openreader/src/queryBuilder.ts

```diff
@@ -465,9 +485,15 @@ export class QueryBuilder {
                         }
                         break
                     }
-                    case 'list-lookup':
-                        rec[req.alias] = this.toResult(row[req.index], req.children)
+                    case 'list-lookup': {
+                        let rows = row[req.index]
+                        if (rows != null) {
+                            rec[req.alias] = this.toResult(row[req.index], req.children)
+                        } else {
+                            rec[req.alias] = []
```

Comment:
I think that query builder shouldn't specify default values but i haven't found a better place

Another way is to make default values explicit and extend FieldRequest interface with an optional field for default value
Or just iterate over data returned from querybuilder and search for null values which should be replaced by default value

But both variants are quite ugly

Reply:
@eldargab

Reply:
It's perfectly fine.

Please, reverse `rows != null` to `rows == null`